src/openforms/forms/tests/selenium_tests/test_form_designer.py

```diff
@@ -0,0 +1,257 @@
+import time
+from contextlib import contextmanager
+
+from django.test import override_settings
+from django.urls import reverse
+
+from furl import furl
+from selenium.common import NoSuchElementException
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+from openforms.accounts.tests.factories import SuperUserFactory
+from openforms.tests.selenium_tests.base import SeleniumTestCase
```

Comment:
`base` is a first in the codebase. But "utils" feels off too... Maybe move it to __init__.py. Meh. Base it is...